createWorkerTransport.js

```diff
@@ -0,0 +1,14 @@
+const createNodeWorkerTransport = (worker) => {
+  return () => {
+    return {
```

Comment:
Can we move constructing the worker here, then we can pass down the URL as a `argv` to the `Worker` constructor. That should make it easy to pass down the url right? (since the function returning should get url as an option).